amr-wind/ocean_waves/relaxation_zones/RelaxationZones.H

```diff
@@ -18,6 +18,8 @@ struct RelaxZonesBaseData
 
     amrex::Real water_depth{0.5};
 
+    amrex::Real g{9.81};
```

Comment:
do we need another variable that carries gravity? Can we hijack it from somewhere else?

Reply:
interested in what you think of my approach. Right now, gravity doesn't belong to any specific part of the code. It is read in with the parser prefix "incflo", but basically each part of the code that uses it parses it directly. I'm trying to get away from that, so in my own development efforts related to waves I have created a gravity variable that belongs to the MultiPhase physics class, and other parts of the code that need gravity get it from that class. However, in the RelaxationZones structure, the functions that need gravity don't have access to the sim variable and therefore don't have access to physics_manager or the MultiPhase class. This g variable is to store the value so it is accessible in the functions where it's needed, but the value is obtained through the MultiPhase class.

Reply:
yeah that seems like a fine solution